src/version.h

```diff
@@ -4,15 +4,58 @@
 #ifndef BITCOIN_VERSION_H
 #define BITCOIN_VERSION_H
 
-#include "clientversion.h"
+#if defined(HAVE_CONFIG_H)
+#include "config/gridcoin-config.h"
+#else
+
+//! Copyright string used in Windows .rc files
+#define COPYRIGHT_STR "2009-"
+
+//These need to be macros, as version.cpp's and bitcoin-qt.rc's voodoo requires it
+#define CLIENT_VERSION_MAJOR       3
+#define CLIENT_VERSION_MINOR       6
+#define CLIENT_VERSION_REVISION    3
+#define CLIENT_VERSION_BUILD       17
+
+#define CLIENT_VERSION_IS_RELEASE false
```

Comment:
Where is this going to be used.
I added a code that if CLIENT_VERSION_BUILD==0 then it is a release, otherwise dev.
This should be made consistent.

Reply:
Agreed, removed.